### PR TITLE
fix: remove stutter, shorter name to allow longer overall name

### PR DESCRIPTION
**Pull request by spring1843**
Fixes # 

**Description**

**How was this change tested?**

*

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: test/suites/common/cleanup.yaml [hunk 1]
```diff
@@ -22,5 +22,5 @@ spec:
   - name: delete-iam-policies
     image: public.ecr.aws/karpenter-testing/tools:latest
     script: |
-      aws cloudformation delete-stack --stack-name KarpenterIAM-$(params.cluster-name)
-      aws cloudformation wait stack-delete-complete --stack-name KarpenterIAM-$(params.cluster-name)
+      aws cloudformation delete-stack --stack-name IAM-$(params.cluster-name)
```
Comment (ellistarn) [inline test/suites/common/cleanup.yaml:25]: Thoughts on prefixing with cluster-name so it sorts well in the UI? Prefixing with IAM always confuses me when I'm trying to delete things.

Comment (ellistarn) [reply test/suites/common/cleanup.yaml:25]: Looking at the names
```
eksctl-karpenter-tests-drift-periodic-mq98t-cluster
eksctl-karpenter-tests-interruption-periodic-n4fqp-cluster
eksctl-karpenter-tests-ipv6-periodic-wq9gq-cluster
eksctl-karpenter-tests-consolidation-periodic-tvb8r-cluster
KarpenterIAM-karpenter-tests-chaos-periodic-tb2wb
KarpenterIAM-karpenter-tests-upgrade-periodic-jgz27
KarpenterIAM-karpenter-tests-utilization-periodic-44xp4
KarpenterIAM-karpenter-tests-drift-periodic-mq98t
KarpenterIAM-karpenter-tests-interruption-periodic-n4fqp
KarpenterIAM-karpenter-tests-integration-periodic-gmnht
KarpenterIAM-karpenter-tests-consolidation-periodic-tvb8r
eksctl-karpenter-tests-utilization-jh6w7-cluster
eksctl-karpenter-tests-ipv6-suitetfg5v-cluster
eksctl-karpenter-tests-drift-xzplh-cluster
eksctl-karpenter-tests-upgrade-suite2fcnj-cluster
eksctl-karpenter-tests-upgrade-suite7tv8f-cluster
eksctl-karpenter-tests-utilization-tswcv-cluster
eksctl-karpenter-tests-upgrade-suitenf4sq-cluster
eksctl-karpenter-tests-ipv6-suitepmhjb-cluster
eksctl-karpenter-tests-upgrade-suite4glp5-cluster
eksctl-karpenter-tests-utilization-vb8cw-cluster
eksctl-karpenter-tests-utilization-drg2g-cluster
eksctl-karpenter-tests-chaos-25zbh-cluster
eksctl-karpenter-tests-ipv6-run-1672863793400-r-sh47x-addon-iamserviceaccount-karpenter-karpenter
eksctl-karpenter-tests-ipv6-run-1672863793400-r-sh47x-addon-aws-ebs-csi-driver
eksctl-karpenter-tests-ipv6-run-1672863793400-r-sh47x-nodegroup-karpenter-tests-ipv6-run-1672863793400-r-sh47x-system-pool
eksctl-karpenter-tests-ipv6-run-1672863793400-r-sh47x-addon-vpc-cni
eksctl-karpenter-tests-ipv6-run-1672863793400-r-sh47x-cluster
KarpenterIAM-karpenter-tests-ipv6-run-1672863793400-r-sh47x
eksctl-karpenter-tests-ipv6-run-1672863793400-cluster
```

Comment (ellistarn) [reply test/suites/common/cleanup.yaml:25]: maybe suffixing with IAM is better?

Comment (spring1843) [reply test/suites/common/cleanup.yaml]: yup I think `iam` lower cased will make this list look better

**Pull request merged**